### ISSUE TITLE
How can I toggle tilda under wayland?

### ISSUE THREAD

**Issue by alexboutros** [lanoxx/tilda#487]
Is there a simple way to get Tilda to toggle using a global shortcut under wayland? I tried using
```
    dbus-send --session --type=method_call \
              --dest=com.github.lanoxx.tilda.Actions0 \
              /com/github/lanoxx/tilda/Actions0 \
              com.github.lanoxx.tilda.Actions.Toggle
```
But nothing happens. 

I ran 
`dbus-send --print-reply --dest=org.freedesktop.DBus  /org/freedesktop/DBus org.freedesktop.DBus.ListNames`
but I am not getting any dbus service for Tilda.

**Comment by vredesbyyrd** [comment]
Just to be sure - did you start your tilda instance with the `--dbus` flag?

**Comment by alexboutros** [comment]
> Just to be sure - did you start your tilda instance with the `--dbus` flag?

I think I was, but it was almost a week ago. When I try to run it with that flag now I'm getting `Unknown option --dbus` from tilda.

**Comment by alexboutros** [comment]
Compiling it did the trick.

**Comment by vredesbyyrd** [comment]
I was just commenting that right now, I believe dbus support is still considered "experimental" and not in a release version yet. Glad it worked out.

**Comment by suhail-singh** [comment]
@lanoxx would it be possible to merge https://github.com/lanoxx/tilda/tree/feature/man-page into `master` and create something like a `1.6-alpha`  release (or a `1.6-alpha` tag at the very least)?

if creating a release is currently not ideal/not feasible, having the `feature/man-page` branch being merged into `master` would still be helpful. if help is needed, i may be able to contribute.

**Comment by lanoxx** [comment]
The DBus support and the `--dbus` option are in main but not yet in a stable release. I usually try to get a release ready for a newer Ubuntu LTS, but i lacked the time doing this for 22.04. I will try to get a release out before Ubuntu 24.04 is going into the Debian import freeze in February.

@suhail-singh, I merged `feature/man-page` just now and a few other changes.

**Comment by lanoxx** [comment]
@vredesbyyrd, the DBus support should work well as far as I am aware, its just that is only available on master and not yet part of a release. The "experimental" part is that we currently need to fiddle with `dbus-send` to toggle the terminal, and it would be nice to have some kind of convenient helper inside tilda to trigger a toggle, which is a bit crude. This was actually already in the man page on main and included in commit 3371b2d74750b2ba370be2f124b428577e0a9495.

The is an open pull request, which I will hopefully get reviewed and merged before the next release that should help smooth the DBus integration a bit: https://github.com/lanoxx/tilda/pull/495

**Comment by lanoxx** [comment]
I have tagged tilda 2.0.0 and uploaded a new release to Debian unstable. It should migrate to testing soon and will then sync to Ubuntu 24.04.
